### PR TITLE
chore: strip out injected script from protocol logs

### PR DESCRIPTION
**Pull request by dgozman**

### CODE REVIEW

File: src/chromium/crConnection.ts [hunk 1]
```diff
@@ -60,7 +60,10 @@ export class CRConnection extends platform.EventEmitter {
     if (sessionId)
       message.sessionId = sessionId;
     const data = JSON.stringify(message);
-    this._debugProtocol('SEND ► ' + data);
+    if (message.method === 'Runtime.evaluate' && message.params && message.params.expression && message.params.expression.includes('src/injected/injected.ts'))
```
Comment (yury-s) [inline src/chromium/crConnection.ts]: Extract in an utility function?

**Review by yury-s — APPROVED** [review]

**Comment by JoelEinbinder** [comment]
I don't like this. I want to see the real protocol logs.

**Comment by yury-s** [comment]
I agree that seeing raw protocol traffic is important but the injected script makes the output very hard to read for me. I'd say make this filter optional but I don't quite like options like that polluting the codebase.

**Comment by JoelEinbinder** [comment]
I'm fine with landing this if it has a very large comment explaining what we are doing here and why.

File: src/chromium/crConnection.ts (hunk 1 above)
Comment (dgozman) [reply src/chromium/crConnection.ts]: Sure.

**Review by pavelfeldman — APPROVED** [review]

**Pull request merged**